ReportingServicesTools.psm1

```diff
@@ -1,2 +1,17 @@
+﻿<#
+    .NOTES
+    --------------------------------------------------------------------------------
+     Code generated by:  SAPIEN Technologies, Inc., PowerShell Studio 2017 v5.4.135
+     Generated on:       05.02.2017 14:43
+     Generated by:       Friedrich Weinmann
```

Comment:
Actually, that's an automatic entry my tool added on the final push, which it shouldn't have. Thanks for pointing it out, removed for my next release (I'll not push that one until I've implemented all change requests - a bit awkward if I push it and deprecate all review comments as I did the last time without realizing that would be the consequence).

For your info though:
That was a small credits info that PSS added. It was _not_ a license notification. All code generated by it is fully my own possession to do with as I please (including publishing it to open source projects).